Action/Configuration/ActionConfiguration.php

```diff
@@ -100,7 +100,7 @@ public function configureOptions(OptionsResolver $resolver)
                 'xls'
             ]);
 
-        // entity will be retrived with this order. It should be an array of field/order mapping
+        // entity will be retrieved with this order. It should be an array of field/order mapping
```

Comment:
retrieved entities are sorted according to this option
it should be an array indexed by the field name with its order as a value